src/components/Quiz.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Quiz extends Component {
+  constructor() {
+    super();
+    this.state = {
+      quizzes: [],
+      questions: [],
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/quizzes', {
+    })
+    .then((response) => {
+      this.setState({
+        quizzes: response.data.quizzes,
+        questions: response.data.quizzes[0].questions,
+      });
+    })
+    .catch(function () {
+      console.log('request failed');
+    });
+   }
+
+  renderQuizTitleToPage(state) {
+    return state.map(q => <article className='quiz-title' key={q.id}>
+                          <h1>{q.title}</h1>
+                        </article>)
+  }
+
+  renderQuestionsToPage(state) {
+    return state.map(q => <article className='question-answers' key={q.id}>
```

Comment:
Questions would be a good thing to break out into it's own component, so you can loop through all the questions and just render something like

```javascript
<Question answers={question.answers} title={question.title} key={question.id}
```